src/filesystem/impls/filer/lib/HTMLRewriter.js

```diff
@@ -103,6 +108,33 @@ define(function (require, exports, module) {
         Array.prototype.forEach.call(elements, rewritePath);
     };
 
+    HTMLRewriter.prototype.scripts = function() {
+        var elements = this.doc.querySelectorAll('script');
+
+        function rewritePath(element) {
+            // Skip any links for protocols (we only want relative paths)
+            var path = element.getAttribute('src');
+            if(!Content.isRelativeURL(path)) {
+                if(element.getAttribute('data-brackets-id') && !jsEnabled) {
+                    element.setAttribute('type', 'text/x-scripts-disabled');
+                } else if (element.getAttribute('data-brackets-id') && jsEnabled) {
+                    if(element.getAttribute('type') === 'text/x-scripts-disabled') {
+                        element.removeAttribute('type');
+                    }
+                }
+
+                return;
```

Comment:
OK, sounds good.  When you split these two steps out, we'll solve this for the non-relative case, even if you want to disable.